tests/Symfony/Upgrade/Test/Fixer/FormGetnameToGetblockprefixFixerTest.php

```diff
@@ -12,10 +12,48 @@ public function testFix($expected, $input, $file)
         $this->makeTest($expected, $input, $file);
     }
 
+    /**
+     * @dataProvider provideFailingExamples
+     */
+    public function testWrongFix($expected, $input, $file, $unExpected)
+    {
+        $fixedCode = $this->getFixer()->fix($file, $input);
+        $this->assertNotSame($unExpected, $fixedCode);
+        $this->assertSame($expected, $fixedCode);
+    }
+
     public function provideExamples()
```

Comment:
Why we don't just use same pattern as before, just add new `$this->prepareTestCase('case*-output.php', 'case*-input.php'),` linkes here?